Angular/src/app/app.component.ts

```diff
@@ -1,10 +1,25 @@
 import { Component } from '@angular/core';
-
+import { TranslateService } from '@ngx-translate/core';
 @Component({
   selector: 'app-root',
   templateUrl: './app.component.html',
   styleUrls: ['./app.component.scss']
 })
 export class AppComponent {
   title = 'Angular';
+
+  constructor(public translate: TranslateService) {
+    translate.addLangs(['en', 'ga', 'th']);
+
+    // this language will be used as a fallback when a translation isn't found in the current language
+    translate.setDefaultLang('en');
+
+    const browserLang = translate.getBrowserLang();
+    // translate.use(browserLang.match(/en|ga/) ? browserLang : 'en');
```

Comment:
Can we remove this commented code? Were they mainly being used to test things out when you were developing?

Reply:
Sure, I will remove them out but now I already merge the code. So sorry about that